repo_tools_data_schema/repo_tools_data_schema.py

```diff
@@ -99,9 +99,10 @@ def _check(d):
                 Optional('orgs'): [valid_org],
                 Optional('repos'): [valid_repo],
                 Optional('champions'): [existing_person],
+                Optional('branches'): not_empty_string,
```

Comment:
Your webhook code is:
```
for access_branch in commit_rights["branches"]:
```
so it should be a list of not empty strings, which I think is right.

Reply:
Ahhhhh yeah. I kinda didn't parse what this was doing. So just putting the `[not_empty_string]` around suffices?

Reply:
:+1: